parser/Node.h

```diff
@@ -143,6 +143,12 @@ class NodeWithExpr final : public Node {
 
 using NodeVec = InlinedVector<std::unique_ptr<Node>, 4>;
 
+inline NodeVec NodeVec1(std::unique_ptr<Node> node) {
+    NodeVec result;
+    result.emplace_back(std::move(node));
+    return result;
```

Comment:
```suggestion
    return NodeVec{1, move(node)};
```

This is short enough that I don't know whether it's worth a separate helper, but up to you.